chroma-manager/tests/integration/shared_storage_configuration/test_conf_params.py

```diff
@@ -14,7 +14,10 @@ def _create_with_params(self):
         # check for it right away and blow up if it's not as we expect.
         self.assertEqual([h['address'] for h in self.hosts], host_addresses)
 
-        volumes = self.wait_for_shared_volumes(4, 2)
+        volumes = self._fetch_help(lambda: self.wait_for_shared_volumes(4, 2),
```

Comment:
Does this still need to be here?

Reply:
will remove when confident it's not needed

Reply:
removed